version/version.go

```diff
@@ -1,5 +1,5 @@
 package version
 
 var (
-	Version = "1.1.0"
+	Version = "1.0.0"
```

Comment:
I wouldn't rollback the version back.

Reply:
For the release process, are we looking to have this file updated manually as well?

Reply:
Yeah, but best thing would be to update the file automatically during the release process (same for the yaml files), I'd like to avoid manual edits as much as possible. We should track this effort under another issue.
So the version in the repo in `master` should imho follow the latest release, ie. `1.1.0`